FrameworkSystem/Client/ComponentInstaller.py

```diff
@@ -184,6 +184,12 @@ def loadDiracCfg( self, verbose = False ):
     self.mysqlMyCnf = os.path.join( self.mysqlDir, '.my.cnf' )
 
     self.mysqlStartupScript = os.path.join( rootPath, 'mysql', 'share', 'mysql', 'mysql.server' )
+    # With the new versions of MySQL server the startup script is in a different location
+    if not os.path.exists( self.mysqlStartupScript ):
+      startupScript = os.path.join( rootPath, 'mysql', 'support-files', 'mysql.server' )
+      if os.path.exists( startupScript ):
+        os.makedirs( os.path.dirname( self.mysqlStartupScript ) )
+        shutil.copy( startupScript, self.mysqlStartupScript )
```

Comment:
Actually, I found the way to place the startup scripts in the same location while compiling Externals. So, I will revert back this change